### PR TITLE
Gh/ts/patch 011 service statistics constants

### PR DESCRIPTION
**Pull request by garyhuntddn**

### CODE REVIEW

**Comment by saghul** [comment]
Converted to draft since there are some merge shenanigans.

File: modules/RTC/TPCUtils.js [hunk 1]
```diff
@@ -239,11 +239,17 @@ export class TPCUtils {
         const track = localTrack.getTrack();
 
         if (isInitiator) {
+            const streams = [];
+
+            if (localTrack.getOriginalStream()) {
```
Comment (saghul) [inline modules/RTC/TPCUtils.js:244]: This looks like an unrelated change, merge sehanigans?

**Comment by garyhuntddn** [comment]
Yeah it must be - grrrr

**Comment by garyhuntddn** [comment]
Cleaned up as a simple export - no point in it being an enum as it's a lonely constant

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by garyhuntddn** [comment]
@saghul quick bump as the merge issues are fixed

**Review by saghul — APPROVED** [review]
